converter/dicom-cast/samples/templates/default-azuredeploy.json

```diff
@@ -94,6 +94,20 @@
             "metadata": {
                 "description": "The endpoint of the FHIR server."
             }
+        },
+        "enableTableStorage": {
```

Comment:
Do we want to expose these has params? Or should we just have the default behavior of, enableTableStorage: True and set the IgnoreSyncOfInvalidTags: true in the container.

Reply:
Basically reduce the settings here to get the default dicom-cast deployment.

Reply:
Updated to set tablestorage to true and leave IgnoreSyncOfInvalidTags as configurable